website/docs/workload-resources.md

```diff
@@ -0,0 +1,79 @@
+---
+id: workload-resources
+title: Working with Workload Resources
```

Comment:
This focuses very much on validation. Do we also want to add something for mutation?

Reply:
Good point -- I personally haven't encountered similar issues with mutations, but that may be because the [mutation library examples](https://github.com/open-policy-agent/gatekeeper-library/tree/master/library/experimental/mutation/pod-security-policy) are still being built out.

Reply:
I think I'm still exploring this question: "Should I mutate my Deployment PodTemplateSpec or just the Pods that it spins up?". Maybe an answer to that question could be an addition to these docs?

Reply:
> I personally haven't encountered similar issues with mutations, but that may be because the mutation library examples are still being built out.

That makes sense. This section talks about how to target resources for mutation: https://open-policy-agent.github.io/gatekeeper/website/docs/mutation#extent-of-changes. Perhaps update this page to scope it to validation only and point to the mutation page for mutation. wdyt?

Reply:
That makes sense. I split the doc into "workload validation" and "workload mutation" sections. A small blurb under "workload mutation" links to the section linked above and has neutral commentary on mutating the Pod or the workload resource PodTemplateSpec.

Reply:
@ritazh do latest updates resolve this comment?